stories/data/configProps.ts

```diff
@@ -1,3 +1,5 @@
+import { CUSTOM_COLOR_PALETTE } from './colors';
```

Comment:
I'm not sure how this is possible, but I think eslint should complain since copyright is not the first line in the file